content/docs/en/getting-started/3-installation.md

```diff
@@ -27,6 +27,18 @@ $ npm install -g nativescript
 
 Verify the installation was successful by running `tns` in your terminal. You should see a list of the available commands.
 
+### Quick Installation
+
+Choose a [template](https://nativescript-vue.org/docs/getting-started/templates/) 
```

Comment:
```suggestion
Create a new project from one of the available [templates](https://nativescript-vue.org/docs/getting-started/templates/). 
```

Consider providing the `tns create` line as part of the commands below as well.

Reply:
The templates page no longer exists, so this part is obsolete.